### PR TITLE
[RMT - 0920] Zhenni QIAN

### PR DESCRIPTION
**Pull request by Jennie-QIAN**

### CODE REVIEW

**Comment by Jennie-QIAN** [comment]
Didn't add the container component cuz I'm not sure where to add that one and why...

File: src/navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                        <CoolButton btnContent="Signup" styles="is-small is-success" />
+                    </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </nav>
+    );
+}
+
```
Comment (In-S4n3) [inline src/navbar/Navbar.js:42]: Nothing to say here. Good job.

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
-  return <h1>Bulma</h1>;
+  return (
+    <>
+      <Navbar />
+      <FormField label="Name" type="text" placeholder="e.g Alex Smith" />
+      <FormField label="Email" type="email" placeholder="e.g. alexsmith@gmail.com" />
+      <Signup />
+      <Message style="is-info" title="Hello World" />
+    </>
+  );
```
Comment (In-S4n3) [inline src/App.js:16]: Super clean, outstanding work. You are just missing the button 1 and button 2. No big deal!

File: src/coolbutton/CoolButton.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import 'bulma/css/bulma.css';
+import React from 'react';
+
+function CoolButton(props) {
+    return (
+        <button className={`button ${props.styles}`}>{props.btnContent}</button>
```
Comment (In-S4n3) [inline src/coolbutton/CoolButton.js:6]: Here you could show the rest of the buttons bulma have looping through the classes as a bonus. Nevertheless. You used it in the navbar and in the signup form. Good job!

File: src/formfield/FormField.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import 'bulma/css/bulma.css';
+import React from 'react';
+
+function FormField(props) {
+    return (
+        <div class="field">
```
Comment (In-S4n3) [inline src/formfield/FormField.js:6]: Super clean! <3

File: src/message/Message.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import 'bulma/css/bulma.css';
+import React from 'react';
+
+function Message(props) {
+    const deleteMessage = () => {
```
Comment (In-S4n3) [inline src/message/Message.js:5]: Awesome!!!

File: src/message/Message.js [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        document.querySelector('article.message').remove();
+    };
+
+    return (
+        <article className={`message ${props.style}`}>
+            <div className="message-header">
+                <p>{props.title}</p>
+                <button className="delete" aria-label="delete" onClick={deleteMessage}></button>
```
Comment (In-S4n3) [inline src/message/Message.js:13]: Once your function is pretty small, you could also used it like this:

`<button
          className="delete"
          aria-label="delete"
          onClick={() => document.querySelector("article.message").remove()}
        ></button>`

and like that, you don't have to create a function above.

**Pull request closed without merging**